docs/workstation_release_management.rst

```diff
@@ -9,23 +9,38 @@ SecureDrop Workstation code spans across two repositories:
 
 The components in the Debian packages are all released together, while the workstation RPM package is released independently.
 
+Communications Process
+-----------------------
+As with SecureDrop server releases, the release manager should work with the Communications Manager
+assigned for the release to prepare announcements that will be shared on the SecureDrop
+blog and on social media after the release is live.
+
+Typically, the kick-off of the QA period is a good time to begin the process. If at all possible
+ensure you leave enough time for review by FPF's editorial team.
```

Comment:
Can we change the implication of this sentence, and instead state that editorial review is expected, and should only be skipped if necessary, like hotfix releases?